README.md

```diff
@@ -1,2 +1,43 @@
 # Personal-Portfolio
+
 This is my Microverse Personal Portfolio.
+
+> Week one Day 3 task
+
+## Built With
+
+- HTML/CSS
+
+## Getting Started
+
+- Click on the clone button to copy the clone
+- Open Terminal and run git clone <copied address>
+- Clone the repo
+- From you terminal run " code ." to open in your code editor.
```

Comment:
[OPTIONAL CHANGES] 
Kindly add how one can clone and get a local version of your repo running. This helps individuals that are not familiar with your code-base to use your project since it is open sourse.